### ISSUE TITLE
Convenience option: --myissues

### ISSUE THREAD

**Issue by pjz** [alisaifee/jira-cli#53]
How about making `jira-cli view --myissues` work ?  It'd be a shortcut for:

`jira-cli view --search-jql='assignee = currentUser() AND resolution = Unresolved order by updated DESC'`

Jira itself has this essentially built-in.

**Comment by AlJohri** [comment]
This would be great. I would even add `--oneline` to this command.

**Comment by alisaifee** [comment]
I've considered adding an option to add custom aliases in the config file. Something along the lines of:
```
[alias]
myissues = view --search-jql='assignee = currentUser() AND resolution = Unresolved order by updated DESC'
```
And then just use it as `jira-cli myissues`

What do you guys think?

**Comment by alisaifee** [comment]
Could you guys look at the linked PR. I'll add documentation for the relevant functionality but basically that will allow for any custom aliases to be defined in the config.

**Comment by AlJohri** [comment]
looks cool. would the aliases be part of `~/.jira-cli/config.cfg` ?

**Comment by alisaifee** [comment]
That is correct - I've added some documentation around configuring the aliases.
